crates/api/src/test_utils.rs

```diff
@@ -218,8 +218,8 @@ pub fn proposer_api_app() -> (
         Arc::new(ChainInfo::for_mainnet()),
         slot_update_sender.clone(),
         Arc::new(ValidatorPreferences::default()),
-        0,
         gossip_receiver,
+        Arc::new(Default::default()),
```

Comment:
This makes tests not compile, it should be
```suggestion
        Default::default(),
```